app/v2/notifications/post_notifications.py

```diff
@@ -182,6 +182,15 @@ def post_bulk():
 
     check_for_csv_errors(recipient_csv, max_rows, remaining_messages)
 
+    for row in recipient_csv.get_rows():
+        try:
+            validate_template(template.id, row.personalisation, authenticated_service, template.template_type)
+        except BadRequestError as e:
```

Comment:
not for this PR but we should have made another exception for this - except TooLongSMSError or somethng